test/mocks/v2alpha1/kube/client/informers/externalversions/factory.go

```diff
@@ -194,25 +194,25 @@ func (f *sharedInformerFactory) InformerFor(obj runtime.Object, newFunc internal
 //
 // It is typically used like this:
 //
-//   ctx, cancel := context.Background()
-//   defer cancel()
-//   factory := NewSharedInformerFactory(client, resyncPeriod)
-//   defer factory.WaitForStop()    // Returns immediately if nothing was started.
-//   genericInformer := factory.ForResource(resource)
-//   typedInformer := factory.SomeAPIGroup().V1().SomeType()
-//   factory.Start(ctx.Done())          // Start processing these informers.
-//   synced := factory.WaitForCacheSync(ctx.Done())
-//   for v, ok := range synced {
-//       if !ok {
-//           fmt.Fprintf(os.Stderr, "caches failed to sync: %v", v)
-//           return
-//       }
-//   }
+//	ctx, cancel := context.Background()
```

Comment:
why did our templates change their spacing here?

Reply:
I think because of the go version bump.